test/types.test.ts

```diff
@@ -187,8 +167,9 @@ Deno.test({
         const file = new InputFile({ url: "https://grammy.dev" });
         const data = await file.toRaw();
         if (data instanceof Uint8Array) throw new Error("no itr");
+
         assertRejects(
-            () => readAll(readerFromIterable(data)),
+            () => convertToUint8Array(data),
```

Comment:
Hey, bro, here no exception will be thrown, I don't know what the issue is,  perhaps it's the ReadableStream.from above?